Marlin/src/lcd/e3v2/jyersui/dwin.cpp

```diff
@@ -2212,7 +2221,7 @@ void CrealityDWIN::menuItemHandler(const uint8_t menu, const uint8_t item, bool
               drawFloat(thermalManager.temp_bed.pid.p(), row, false, 100);
             }
             else {
-              modifyValue(thermalManager.temp_bed.pid.Kp, 0, 5000, 100, thermalManager.updatePID);
```

Comment:
Removed because `thermalManager.updatePID` does not exist when `PIDTEMP` is not set. It seems only to affect hotend PID anyway and I couldn't find anything similar that works on heated bed PID.

Reply:
The equivalent for the bed is `temp_bed.pid.reset()`.